src/OrchardCore/OrchardCore.Redis.Abstractions/RedisOptions.cs

```diff
@@ -2,16 +2,16 @@
 
 namespace OrchardCore.Redis
 {
-    /// <summary>
-    /// Redis options.
-    /// </summary>
     public class RedisOptions
```

Comment:
```suggestion
    /// <summary>
    /// Configuration options for <see cref="IRedisService"/>.
    /// </summary>
    public class RedisOptions
```

As done here https://github.com/dotnet/aspnetcore/blob/39219394f66854db5d556ad78f9bb065ee48b585/src/Caching/StackExchangeRedis/src/RedisCacheOptions.cs#L12-L15